docs/interfaces/action-service.txt

```diff
@@ -0,0 +1,70 @@
+// Actions Service Interface
+
+/**
+ * Executes multiple actions in the context of the user given by the user_id.
+ * There are two modes of execution:
+ * atomic=false (default):
+ *   All actions are validated in common, so if one action or one payload of
+ *   one action fails, the request is aborted with an ActionException indicating
+ *   the problematic action with both indices.
+ * atomic=true:
+ *   Each action is validated by it's own. If there is an error, the error must
+ *   be reported via an ActionError in the ActionsResponse. The actions result
+ *   is not written into the Datastore. It might raise an ActionException if the
+ *   single write request to the Datastore fails (e.g. because of locking there)
+ *
+ * For general, non specific action-related, errors an ActionException is used.
+ *
+ * @throws ActionException
+ */
+handle_request(payload: Action[], user_id: Id): ActionsResponse
```

Comment:
The atomic flag should be here and not inside the Action interface.

Reply:
@jsangmeister I think norman is right. ok?

Reply:
Yes